src/test/app/forms/models/address.ts

```diff
@@ -2,20 +2,38 @@
 /* tslint:disable:no-unused-expression */
 
 import { expect } from 'chai'
-import * as _ from 'lodash'
 import { Validator } from 'class-validator'
 
-import { expectValidationError } from './validationUtils'
-import { Address, ValidationErrors as AddressValidationErrors } from 'forms/models/address'
-import { CorrespondenceAddress, ValidationErrors as CorrespondenceAddressValidationErrors } from 'forms/models/correspondenceAddress'
+import { expectValidationError, generateString, evaluateErrorMsg } from './validationUtils'
+import {
+  Address,
+  ValidationErrors as AddressValidationErrors,
+  ValidationConstants as AddressValidationConstants
+} from 'forms/models/address'
+import {
+  CorrespondenceAddress,
+  ValidationConstants as CorrespondenceAddressValidationConstants,
+  ValidationErrors as CorrespondenceAddressValidationErrors
+} from 'forms/models/correspondenceAddress'
+import { Address as ClaimAddress } from 'claims/models/address'
 
 describe('Address/CorrespondenceAddress', () => {
   [
-    { classFunction: Address, validationErrors: AddressValidationErrors },
-    { classFunction: CorrespondenceAddress, validationErrors: CorrespondenceAddressValidationErrors }
+    {
+      classFunction: Address,
+      validationErrors: AddressValidationErrors,
+      validationConstants: CorrespondenceAddressValidationConstants
```

Comment:
It does not really matter from runtime perspective, but it should be `AddressValidationConstants` here and `CorrespondenceAddressValidationConstants` at line 30.

Reply:
fixed